osu.Game/osu.Game.csproj

```diff
@@ -18,11 +18,13 @@
     </None>
   </ItemGroup>
   <ItemGroup Label="Package References">
+    <!-- WorkaroundWorkaround for a System.IO.FileNotFoundException on startup. Required because of old EF Core version. -->
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="2.2.0" />
     <PackageReference Include="Dapper" Version="2.0.35" />
     <PackageReference Include="DiffPlex" Version="1.6.3" />
     <PackageReference Include="Humanizer" Version="2.8.26" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.1.9" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="3.1.9" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="5.0.0" />
```

Comment:
will need re-testing of mobile platforms as we have them pinned to a much earlier version for compatibility reasons.

Reply:
These changes were reverted, no longer an issue.